docs/rundown.rst

```diff
@@ -64,6 +102,46 @@ and two small playbooks under a ``playbooks`` directory.
    root of your project. ``ansible-sign project`` commands, as a rule, always
    take the project root directory as their last argument, thus we will simply
    use ``.`` to indicate the current Working Directory.
+Generating a GPG Key
+===================
+
+Full documentation:
+https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key
```

Comment:
This might be okay to link to, but I'd like someone else to sign off on that before we do it. There's a RH blog post about making keys and I link to it at the very start of this file.